### PR TITLE
GobalMetadata doesn't return error

### PR DESCRIPTION
**Pull request by kopiczko**
All implementations of MetadataService.GlobalMetadata() never fail.

### CODE REVIEW

File: cmd/torusblk/nbd.go [hunk 1]
```diff
@@ -97,18 +97,16 @@ func connectNBD(srv *torus.Server, f *block.BlockFile, target string, closer cha
...
-	if err != nil {
-		return err
-	}
+	gmd := srv.MDS.GlobalMetadata()
 
 	handle := nbd.Create(f, int64(size), int64(gmd.BlockSize))
 
 	if target == "" {
-		target, err = nbd.FindDevice()
+		t, err := nbd.FindDevice()
```
Comment (nak3) [inline cmd/torusblk/nbd.go]: Why this change (`target`->`t`) is necessary? convention or something?

**Comment by barakmich** [comment]
Hm. This is a side-effect of the fact that, upon creation of a Metadata object, if it can't get the GMD, then it can't be created. This PR would canonicalize that statement.

I'm okay with this notion; if we add a comment to this effect on the interface.

**Comment by kopiczko** [comment]
I've added comment to the method and commit message. I tried to make that clear in as few words as possible.

File: cmd/torusblk/nbd.go (hunk 1 above)
Comment (kopiczko) [inline cmd/torusblk/nbd.go]: Note err had to be initialized here (see the colon). `target, err := nbd.FindDevice()` would shadow the target variable from the outer scope, and its value would never be changed. It could also be resolved by declaring err at the first place, but I'm not sure whether that is clearer.

Comment (nak3) [inline cmd/torusblk/nbd.go]: Thank you for your clarification. I'm sorry, I missed colon and removing the error initialization lines  above.
